### PR TITLE
Fix Quick Look when window is inactive on completion

### LINKED ISSUE #100: Quick Look doesn't work if the window is not focused
Quick Look doesn't work if the window is not focused (main/key) when the "conversion completed" screen is presented. Reproducible on macOS 10.14.6, but not on macOS 10.15 beta 3. See https://github.com/sindresorhus/Gifski/pull/94#issuecomment-512754498.

### PR DESCRIPTION
**Pull request by sindresorhus**
Fixes #100

---

With focus:

```
...
Responder chain:
(Main view controller: Optional(<Gifski.ConversionViewController: 0x600003b02220>)
  - <Gifski.ConversionViewController: 0x600003b02220>
  - <NSWindow: 0x600003e11300>
  - <Gifski.MainWindowController: 0x600002109ef0>
Responder chain:
(Main view controller: Optional(<Gifski.ConversionCompletedViewController: 0x10212bcf0>)
  - <Gifski.ConversionCompletedViewController: 0x10212bcf0>
  - <NSWindow: 0x600003e11300>
  - <Gifski.MainWindowController: 0x600002109ef0>
```

Without focus:

```
...
Responder chain:
(Main view controller: Optional(<Gifski.ConversionViewController: 0x600003b06680>)
  - <Gifski.ConversionViewController: 0x600003b06680>
  - <NSWindow: 0x600003e02600>
  - <Gifski.MainWindowController: 0x60000212d180>
```

Notice how the `ConversionCompletedViewController` never becomes first responder when the window is not active.

---

I'm gonna remove the debug utilities before merging.

### CODE REVIEW

File: Gifski/util.swift [hunk 1]
```diff
@@ -2449,7 +2449,6 @@ extension NSViewController {
 		}, completionHandler: {
 			window.contentViewController = viewController
 			viewController.view.animator().alphaValue = 1.0
-			window.makeFirstResponder(viewController)
```
Comment (sindresorhus) [inline Gifski/util.swift]: @sunshinejr This was the culprit. Why did you add it?

File: Gifski/util.swift [hunk 2]
```diff
@@ -2449,7 +2449,6 @@ extension NSViewController {
 		}, completionHandler: {
```
Comment (sindresorhus) [inline Gifski/util.swift]: Unrelated, but why are we not using https://developer.apple.com/documentation/appkit/nsviewcontroller/1434431-present? We could use it with a custom animator: https://stackoverflow.com/a/28493373/64949

File: Gifski/util.swift (hunk 1 above)
Comment (sunshinejr) [reply Gifski/util.swift]: Really interesting. So when I pushed the VC it wouldn't be the first responder so I had to do it manually.

File: Gifski/util.swift (hunk 2 above)
Comment (sunshinejr) [reply Gifski/util.swift]: No particular reason, found that content switching API and used it.

File: Gifski/ConversionCompletedViewController.swift [hunk 3]
```diff
@@ -41,6 +41,9 @@ final class ConversionCompletedViewController: NSViewController {
 	override func viewDidAppear() {
 		super.viewDidAppear()
 
+		self.view.window?.makeFirstResponder(self)
```
Comment (sunshinejr) [inline Gifski/ConversionCompletedViewController.swift]: really interesting: so making it a first responder in `viewDidAppear` is working but the generic way in `AppDelegate` with the app became active method is not? In that case we might need to see if other VCs also need to do that in `viewDidAppear`. Really strange.

Comment (sindresorhus) [reply Gifski/ConversionCompletedViewController.swift]: Yes, the problem seems to be making them first responder before their view is shown. Also really strange that it works in Catalina.

**Comment by sunshinejr** [comment]
@sindresorhus after the fix I can't cancel with ESC on conversion VC. I think we should add the `.makeFirstRespoder` to all the VCs on `viewDidAppear`

**Comment by sindresorhus** [comment]
> after the fix I can't cancel with ESC on conversion VC.

Fixed.

> I think we should add the .makeFirstRespoder to all the VCs on viewDidAppear

I don't think we should add it unless it's needed. I couldn't find any other cases it's needed.

File: Gifski/ConversionViewController.swift [hunk 4]
```diff
@@ -42,16 +42,16 @@ final class ConversionViewController: NSViewController {
...
+		view.window?.makeFirstResponder(self)
 
 		start(conversion: conversion)
 	}
 
 	/// Gets called when the Esc key is pressed.
-	/// Reference: https://stackoverflow.com/a/42440020
-	@objc
-	func cancel(_ sender: Any?) {
+	override func cancelOperation(_ sender: Any?) {
```
Comment (sindresorhus) [inline Gifski/ConversionViewController.swift]: I also changed this as with the macOS 10.14 SDK, NSViewController has `cancelOperation` too.

**Pull request merged**